draft-ietf-rats-eat.md

```diff
@@ -818,6 +818,50 @@ uptime-claim = (
 )
 ~~~~
 
+## The Context Claim (context)
+
```

Comment:
What about renaming this to "Intended Use" or "Intended Use Class" or "Use Class"? It definitely seems like a classification scheme so I think of the different values as different *classes*.

Reply:
Changed name from 'context' to 'intended-use'